style.css

```diff
@@ -83,3 +82,322 @@ h2 {
+}
+
+.projects h3 {
+  width: 295px;
+  height: 81px;
+  left: 16px;
+  top: 8px;
+  font-family: Inter, "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-top: 12px;
+  margin-bottom: 12px;
+  z-index: 3;
+}
+
+.projects .Feat-text {
+  width: 260px;
+  left: 16px;
+  top: 101px;
+  font-family: Inter, "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 22px;
+  color: #fff;
+  margin: 1px 10px;
+  z-index: 2;
+}
+
+.proj-container {
+  display: grid;
+  grid-template-rows: repeat(3, 1fr);
+}
+
+.about-section {
+  background-image: url(src/Group.svg);
+  background-repeat: no-repeat;
+  background-color: #1c1a19;
+  background-position: bottom right;
+  width: 100vw;
+  height: 100vh;
+  top: 4137px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  align-content: center;
+}
+
+.about-section h2 {
+  color: #ff6b00;
+  font-family: Inter, "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 72px;
+  line-height: 88px;
+  letter-spacing: 0.37px;
+}
+
+.about-section p {
+  font-family: Inter, "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.skills {
+  background-color: #1c1a19;
+  width: 100vw;
+  height: 100vh;
+  top: 4137px;
+  color: #f4f5f7;
+  display: grid;
+  grid-template-rows: repeat(3, 1fr);
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-evenly;
+  align-content: center;
+}
+
+.key-skills {
+  width: 114px;
+  height: 24px;
+  top: 12px;
+  font-family: Inter, "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 22px;
+  line-height: 43px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+}
+
+.key-skills h3 {
+  width: 227px;
+  height: 24px;
+  left: 42px;
+  top: 8px;
+  font-family: Inter, "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  padding: 10px;
+}
+
+.skills-macro {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-content: center;
+  width: 100%;
+  height: 100%;
+  align-items: center;
+  text-align: center;
+}
+
+.skills-feat {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-content: center;
+  top: 12px;
+  font-family: Inter, "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 18px;
+  line-height: 43px;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  padding: 20px 0;
+}
```

Comment:
I highly recommend that your design strictly matches the figma design template that you choose to work with. I know you have already done a great job 💪🏻 and I applaud you for it 🥂. I suggest that you add a little bit of effort and bring the specified design to life 😄.

- [ ] I also request that you focus and import the right font-family and font-weights as used in the design files. 

| Your design | Figma design |
| --- | --- |
| <img width="307" alt="let" src="https://user-images.githubusercontent.com/4247740/143672829-3e10f98d-842d-4c91-90e3-ba8536d56303.PNG"> | <img width="284" alt="let5" src="https://user-images.githubusercontent.com/4247740/143673082-4025dd2a-0895-4b42-add3-8edd8b7b6622.PNG"> |
| <img width="302" alt="let2" src="https://user-images.githubusercontent.com/4247740/143672875-7717e3ed-7b85-4dc6-bb28-bffbd3b79060.PNG"> | <img width="283" alt="let6" src="https://user-images.githubusercontent.com/4247740/143673068-784ec4c5-55f5-4f30-9ae9-797424a67007.PNG"> |
| <img width="302" alt="let3" src="https://user-images.githubusercontent.com/4247740/143672903-d924ba77-5af3-400a-9d1c-f5592c0c4215.PNG"> | <img width="283" alt="let7" src="https://user-images.githubusercontent.com/4247740/143673045-14abe0e6-345c-41b8-878f-7f689dcf65ac.PNG"> |
| <img width="300" alt="let4" src="https://user-images.githubusercontent.com/4247740/143672962-32c42eb3-5ff2-4fc4-88ed-12d5ef36da9e.PNG"> | <img width="283" alt="let8" src="https://user-images.githubusercontent.com/4247740/143673035-826026a8-897e-4306-aa20-a233b5f0dcf8.PNG"> |